pkg/repart/disk_repart.go

```diff
@@ -217,7 +217,6 @@ func runSystemdRepart(s *sys.System, target string, parts []Partition, flags ...
 	args = append(args, target)
 
 	out, err := s.Runner().RunEnv("systemd-repart", []string{"PATH=/sbin:/usr/sbin:/usr/bin:/bin"}, args...)
-	s.Logger().Debug("systemd-repart output:\n%s", string(out))
```

Comment:
@davidcassany I'm not familiar with repart output. Do we want to keep it on successful operations?